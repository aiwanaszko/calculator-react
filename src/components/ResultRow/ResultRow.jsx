import React from "react";
import "./ResultRow.scss";

export default function ResultRow(props) {
  return (
    <div className="result-row">
      <input type="text" readOnly value={props.value} />
    </div>
  );
}
