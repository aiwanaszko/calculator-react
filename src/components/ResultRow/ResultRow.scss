.result-row input {
  font-family: "Montserrat";
  width: 98%;
  text-align: right;
  background-color: black;
  border: none;
  height: 50px;
  color: white;
  font-size: 36px;
  font-weight: 900;
  outline: none;
  margin-bottom: 30px;
}
