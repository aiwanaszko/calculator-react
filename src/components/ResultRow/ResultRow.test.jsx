import React from "react";
import renderer from "react-test-renderer";
import ResultRow from "./ResultRow";

it("renders the ResultRow correctly", () => {
  const component = renderer.create(<ResultRow />).toJSON();
  expect(component).toMatchSnapshot();
});
