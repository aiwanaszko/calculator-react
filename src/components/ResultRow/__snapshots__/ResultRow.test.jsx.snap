// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the ResultRow correctly 1`] = `
<div
  className="result-row"
>
  <input
    readOnly={true}
    type="text"
  />
</div>
`;
